tests/unit/postTestBuildComment.ts

```diff
@@ -1,18 +1,23 @@
 import * as core from '@actions/core';
 import {when} from 'jest-when';
+import type {Writable} from 'type-fest';
 import ghAction from '../../.github/actions/javascript/postTestBuildComment/postTestBuildComment';
 import GithubUtils from '../../.github/libs/GithubUtils';
 
 const mockGetInput = jest.fn();
-const mockCreateComment = jest.fn();
+const createCommentMock = jest.spyOn(GithubUtils, 'createComment');
```

Comment:
why this is changed?

Reply:
This was changed because we couldn't assign a jest.mock into GithubUtils.createComment, this way we mock the createComment correctly and avoid TS errors.